### ISSUE TITLE
[Screen reader – Make Code - Home Page]: Screen reader announces incorrect information while navigating on the 'setting' menu item.

### ISSUE THREAD

**Issue by monisverma** [microsoft/pxt-microbit#5451]
### **User impact:**
Visually impaired people who depend on screen readers for navigation will get impacted if screen reader does not announce the correct information while navigating on the header section.

### **Note:**
User credentials should NOT be included in the bug.

### **Repro Steps:**

1. Open the URL https://makecode.microbit.org/ 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Navigate on the Setting icon and verify whether correct information is announced by screen reader or not.

### **Actual Result:**
**Issue:** Screen reader announces incorrect information while navigating on the 'setting' menu item.
**Observation:** Jaws is announcing the information as "Dropdown menu more dot dot dot two of three".

### **Expected Result:**
When navigating to the page, Screen reader should announce the correct information while navigating on the 'setting' menu item.
Ex: Screen Reader should announce the information as 'Setting menu item 2 of 3'.

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
ARIA Authoring practices https://www.w3.org/WAI/ARIA/apg/

### **UIA/DOM details available:** 
yes

### **MAS Reference:**
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### **Reference Links:** 

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### **Test Environment:**
**OS:** Windows 11 22H2 (OS Build:22621.3085).
**Browser:** Chrome Version 121.0.6167.86 (Official Build) (64-bit)
**URL:** https://makecode.microbit.org/
**Screen Reader:** JAWS

### **Attachment:**
![Screen reader announces incorrect information while navigating on the 'setting' menu item](https://github.com/microsoft/pxt-microbit/assets/158456802/eebb3623-885b-4a03-81df-d049ad7ed382)

**Comment by monisverma** [comment]
#Accessibility#A11yMAS;#A11ySev2;#E2E-HCL-MakeCode-Jan2024;#MAS1.3.1;#HCL;#A11yAuto;#K4W;#Win11-Chrome;#benchmark

**Comment by vroopea** [comment]
As checked the issue on URL: [https://makecode.microbit.org/](https://makecode.microbit.org/). The issue is still repro and severity is fine. #Regressed:04-08-24;InternalReviewed-Apr2024

**Comment by vroopea** [comment]
As checked the issue on URL: https://makecode.microbit.org/. The issue is still repro and severity is fine. #Regressed:04-08-24;InternalReviewed-Apr2024
